web/modules/custom/farm_nfa/modules/farm_nfa_natural_forest/src/Plugin/Plan/PlanType/Natural.php

```diff
@@ -24,7 +24,7 @@ public function buildFieldDefinitions() {
     $options = [
       'type' => 'entity_reference',
       'label' => $this->t('CFRs'),
-      'description' => $this->t('Select the CFRs that this plan pertains to.'),
+      'description' => $this->t('Select the CFRs that this plan pertains to. If the desired compartment does not yet exist in the system, a new record must be created for it. This can be done via the <a href="@add">Add Compartment</a> form.', ['@add' => '/asset/add/compartment']),
```

Comment:
I think these both need to say "CFR" instead of "compartment".

Reply:
I might be missing something, but IIRC, this is to add CFRs that might have missing compartments? /c @LeScruf

Reply:
It's a CFR reference field, and can only reference CFR assets. But it's directing the user to create compartments.

I think the compartment creation help text might belong in the management form instead of here (which is for the Plan creation form). This would be worth discussing on our next call.